source/config/__init__.py

```diff
@@ -88,17 +86,15 @@ class RegistryKey(str, Enum):
 	"""
 
 
-if version_year < 2023:
+if globalVars._useDeprecatedAPI:
 	RUN_REGKEY = RegistryKey.RUN.value
 	"""
-	Deprecated, for removal in 2023.
-	Use L{RegistryKey.RUN} instead.
+	Deprecated, use L{RegistryKey.RUN} instead.
 	"""
 
 	NVDA_REGKEY = RegistryKey.NVDA.value
 	"""
-	Deprecated, for removal in 2023.
-	Use L{RegistryKey.NVDA} instead.
+	Deprecated, use L{RegistryKey.NVDA} instead.
 	"""

```

Comment:
Using the `__getattr__` approach would allow a warning to be given on access. This would be easier for add-on authors to get a full list of deprecation usages up front (before starting to fix them all). It would give them an indication about how long it will take to fix the usages.